src/server/server_family.cc

```diff
@@ -1626,38 +1626,49 @@ void ServerFamily::FlushAll(CmdArgList args, ConnectionContext* cntx) {
   cntx->SendOk();
 }
 
+bool ServerFamily::DoAuth(ConnectionContext* cntx, std::string_view username,
+                          std::string_view password) const {
+  const auto* registry = ServerState::tlocal()->user_registry;
+  const bool is_authorized = registry->AuthUser(username, password);
```

Comment:
Add `CHECK(registry != nullptr);`?

Reply:
done, but with `CHECK(registry)` :nerd_face:

Reply:
I think that comparing to `nullptr` is a clearer phrasing, but :shrug: